core/tests/protractor.conf.js

```diff
@@ -78,7 +67,7 @@ exports.config = {
   //
   // A base URL for your application under test. Calls to protractor.get()
   // with relative paths will be prepended with this.
-  baseUrl: 'http://localhost:4445',
+  baseUrl: 'http://localhost:9001',
```

Comment:
Mmmh I do not think so, Is 4444 not the universal port that selenium driver run on?